src/platform/symbols.map

```diff
@@ -202,5 +202,8 @@ MIR_PLATFORM_2.8 {
     mir::graphics::DRMFormat::operator?unsigned?int*; /* Is actually operator uint32_t(), but 🤷 */
 
     mir::graphics::drm_modifier_to_string*;
-  };
+ 
+    mir::udev::Device::clone*;
+    mir::udev::Context::char_device_from_devnum*;
+   };
 } MIRPLATFORM_2.7;
```

Comment:
If I'm right about the ABI break, then we should be consolidating a MIRPLATFORM_2.8 stanza.